compiler/ir/ir.tree/src/org/jetbrains/kotlin/ir/descriptors/IrBuiltIns.kt

```diff
@@ -198,6 +198,7 @@ class IrBuiltIns(
 
     val eqeqeqSymbol = defineOperator(OperatorNames.EQEQEQ, bool, listOf(anyN, anyN))
     val eqeqSymbol = defineOperator(OperatorNames.EQEQ, bool, listOf(anyN, anyN))
+    val andandSymbol = defineOperator(OperatorNames.ANDAND, bool, listOf(bool, bool))
```

Comment:
Since this operator is only used in JVM, could you please declare it in `IrIntrinsicMethods`? Similarly to how for example JS operator intrinsics are declared in [`JsIntrinsics`](https://github.com/JetBrains/kotlin/blob/master/compiler/ir/backend.js/src/org/jetbrains/kotlin/ir/backend/js/JsIntrinsics.kt)

Reply:
Addressed.